src/ci/docker/host-x86_64/mingw-check/Dockerfile

```diff
@@ -41,4 +41,4 @@ ENV SCRIPT python3 ../x.py --stage 2 test src/tools/expand-yaml-anchors && \
            /scripts/validate-error-codes.sh && \
            # Runs checks to ensure that there are no ES5 issues in our JS code.
            es-check es6 ../src/librustdoc/html/static/js/*.js && \
-           eslint ../src/librustdoc/html/static/js/*.js
+           eslint -c ../src/librustdoc/html/static/.eslintrc.js ../src/librustdoc/html/static/js/*.js
```

Comment:
I no sure the change of this line is necessary, however, to make it explicit, I prefer to keep this change.

Reply:
I also prefer to keep this change.